dist/definitions.json

```diff
@@ -1014,5 +1014,169 @@
         "$ref": "#/definitions/privacyAgreementAccepted"
       }
     }
+  },
+  "form8940": {
+    "type": "object",
+    "unemployability": {
+      "type": "object",
+      "properties": {
+        "disabilityPreventingEmployment": {
+          "type": "string"
+        },
+        "underDoctorHopitalCarePast12M": {
+          "type": "boolean"
+        },
+        "doctorProvidedCare": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "name": {
+              "type": "string"
+            },
+            "address": {
+              "$ref": "#/definitions/address"
```

Comment:
I am unsure for the address what the reqs are for 8940 form. If the schema uses #/definitions/address then as per 526 all claims schema, required fields are [‘country’, ‘city’, ‘addressLine1’]. However if we use #/definitions/centralMailAddress, then postalCode is required unless you have overridden centralMailAddress to use baseAddressDef